packages/components/src/components/money/money.jsx

```diff
@@ -17,13 +18,22 @@ const Money = ({
 
     // if it's formatted already then don't make any changes unless we should remove extra -/+ signs
     const value = (has_sign || should_format) ? Math.abs(amount) : amount;
-    const final_amount = should_format ? CurrencyUtils.formatMoney(currency, value, true) : value;
-
+    const decimalPart = CurrencyUtils.getDecimalPart(amount);
+    const needsCryptoToggle = CurrencyUtils.isCryptocurrency(currency) &&
+        (decimalPart && decimalPart.length > CurrencyUtils.getDecimalPlaces(currency));
+    let final_amount = null;
+    if (needsCryptoToggle) {
+        final_amount = CurrencyUtils.getCryptoFormat(value);
+    } else {
+        final_amount = should_format ? CurrencyUtils.formatMoney(currency, value, true) : value;
+    }
     return (
         <React.Fragment>
             {has_sign && sign}
-            <span className={classNames(className, 'symbols', `symbols--${currency.toLowerCase()}`)} />
-            {final_amount}
+            <span
+                className={classNames(className, 'symbols', `symbols--${currency.toLowerCase()}`)}
+            />
+            {needsCryptoToggle ? <CryptoMoney {...final_amount} /> : final_amount}
```

Comment:
Passing implicit props makes it harder to debug it in the future. IMO using a direct key=value is way more cleaner.

Reply:
True. Specially in this case where there are just a few keys it really makes the interface clearer.